docs/source/markdown/podman-container-checkpoint.1.md

```diff
@@ -205,12 +206,12 @@ Create a checkpoint image for the container "mywebserver".
 
 Dumps the container's memory information of the latest container into an archive.
 ```
-# podman container checkpoint -P -e pre-checkpoint.tar.gz -l
+# podman container checkpoint -P -e pre-checkpoint.zstd -l
```

Comment:
According to 
https://en.wikipedia.org/wiki/Zstd
the filename extension is `zst`

Another thing: `tar` part is missing

I guess it should be `pre-checkpoint.tar.zst`